serialization/journal.proto

```diff
@@ -1539,6 +1510,75 @@ message NewPlugin {
   optional Return return = 3;
 }
 
+message PartIncrementIntrinsicForce {
+	extend Method {
```

Comment:
The spurious renaming was not great, but changing spaces to tabs is actively harmful.